test/test-helper/cli.ts

```diff
@@ -0,0 +1,47 @@
+import { Writable } from 'node:stream';
+import { BaseContext, Cli, CommandClass } from 'clipanion';
+import { useTempDir } from './use-temp-dir';
+
+
+class CliError extends Error {
```

Comment:
for consistency with the other files i would suggest to do the export inline instead of all at the bottom in a single statement.

```suggestion
export class CliError extends Error {
```